tests/UI/Component/Symbol/Icon/IconTest.php

```diff
@@ -195,18 +173,8 @@ public function testAllStandardIconsExist() : void
             $path = self::ICON_PATH . "icon_" . $icon_abr . ".svg";
             $this->assertTrue(file_exists($path), "Missing Standard Icon: " . $path);
         }
-    }
+        exit;
```

Comment:
@Amstutz This seems to break the Tests in trunk. (Ends the Test run, therefore all tests after this (aprox 500) are inoffially skipped).

Im not sure if there is an solution for this by adaption of the pipeline but i assume the remove of this line should is the best option

Reply:
Good catch you both. Will look into this, @Amstutz is in the holidays.

Reply:
I have removed the thing: https://github.com/ILIAS-eLearning/ILIAS/commit/02c5234effc662cad1cd9b63a83858ae7944c2d1